dsc/workflows/base/__init__.py

```diff
@@ -282,7 +282,8 @@ def submit_items(self, collection_handle: str) -> list:
                 continue
 
             # validate whether a message should be sent for this item submission
-            if not self.allow_submission(item_submission_record):
+            item_submission.populate_from_db()
```

Comment:
A minimally-invasive way to use this method, it will likely need to go into `item_submission_iter`